package.json

```diff
@@ -83,14 +83,15 @@
     "espower-typescript": "^9.0.0",
     "mocha": "^5.2.0",
     "npm-run-all": "^4.1.3",
+    "nyc": "^13.0.1",
     "power-assert": "^1.6.0",
     "rimraf": "^2.6.2",
+    "source-map-support": "^0.5.9",
```

Comment:
that's not a dev dependency. only really useful at runtime. Is that coming from somewhere else?

Reply:
nyc/istanbul needs it for test coverage, see line 113